src/test/java/subway/fixture/LineFixture.java

```diff
@@ -0,0 +1,40 @@
+package subway.fixture;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.MediaType;
+import subway.dto.line.LineRequest;
+
+public class LineFixture {
+    public static ExtractableResponse<Response> createLine(
```

Comment:
fixture 👍 👍 를 활용함으로써 재사용성도 올라가고 가독성도 좋아진 것 같아요.
한가지 더 의견을 드리자면 메서드명을 더 읽기 쉽게 저는 한글로 작성하는 편인데요. 
참고만 부탁드릴께요~
```suggestion
    public static ExtractableResponse<Response> 노선을_생성한다. (
```